coco-format-conversion/coco_format_conversion.py

```diff
@@ -78,7 +78,7 @@ def images_annotations(manga109_root_dir, train_val_cutoff, val_test_cutoff):
              val_test_cutoff else ret_images_test).append(image)
 
         # Annotations
-        frame_bb_list = tuple(bb_iter(title, manga109_root_dir, categories=["frame"]))
```

Comment:
You should change the name of the variable because the category is not limited to "frame".

Reply:
Thanks, I changed the variable names: https://github.com/manga109/manga109-demos/pull/5/commits/07f25d870913c4df9e0ffd911dbb48d73a11d98b#diff-d987251ac87a074cdfe01181aa46be8d0013bd74f9b6f7173205c2097c61ccacR81